sonar-project.properties

```diff
@@ -0,0 +1,19 @@
+sonar.projectKey=vivid-planet_comet
+sonar.organization=vivid-planet
+
+# This is the name and version displayed in the SonarCloud UI.
+#sonar.projectName=comet
+#sonar.projectVersion=1.0
+
+
+# Path is relative to the sonar-project.properties file. Replace "\" by "/" on Windows.
+#sonar.sources=.
+
+# Encoding of the source code. Default is default system encoding
+#sonar.sourceEncoding=UTF-8
```

Comment:
```suggestion
```

I'd remove these comments

Reply:
I've added this because sonar recommends this